tutor/src/screens/assignment-review/index.js

```diff
@@ -134,19 +125,26 @@ class AssignmentReview extends React.Component {
     );
   }
 
+  renderNoStudents() {
+    return (
+      <>
+        <StyledTabs
+          selectedIndex={0}
+          onSelect={this.onTabSelection}
+          tabs={['Details']}
+        />
+        <Details ux={this.ux} />
+      </>
+    );
+  }
+  
```

Comment:
I don't see this function being used anywhere.

Reply:
yep good catch..  I'd added this with an earlier version before we decided to hide the "NoStudents" message.